### PR TITLE
[LocalAuthentication] Update to iOS 10.3 beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/localauthentication.cs [hunk 1]
```diff
@@ -56,19 +56,19 @@ interface LAContext {
...
 		string LocalizedCancelTitle { get; set; }
 
-#if !MONOMAC
-		[Availability (Introduced = Platform.iOS_8_3, Deprecated = Platform.iOS_9_0)]
-		[NullAllowed]
-		[Export ("maxBiometryFailures")]
-		NSNumber MaxBiometryFailures { get; set; }
-
-		[iOS (9,0)]
+		[iOS (9,0)][Mac (10,12,3)]
```
Comment (rolfbjarne) [inline src/localauthentication.cs]: Shouldn't it be `[Mac (10,12,4)]`?

**Comment by VincentDondain** [comment]
Unrelated tests failures (due to the move to Xcode8.3)
apitest - https://bugzilla.xamarin.com/show_bug.cgi?id=51799
monotouch-test - https://bugzilla.xamarin.com/show_bug.cgi?id=51800
introspection-mac -  https://bugzilla.xamarin.com/show_bug.cgi?id=51801

File: src/localauthentication.cs (hunk 1 above)
Comment (spouliot) [inline src/localauthentication.cs]: Also the type is decorated with `onlyOn64 : true` which should be added to any member attribute (others are missing too).

File: src/localauthentication.cs [hunk 2]
```diff
@@ -56,19 +56,19 @@ interface LAContext {
...
 		[Field ("LATouchIDAuthenticationMaximumAllowableReuseDuration")]
 		double /* NSTimeInterval */ TouchIdAuthenticationMaximumAllowableReuseDuration { get; }
 
-		[iOS (9,0)]
+		[iOS (9,0)][Mac (10,12,3)]
```
Comment (spouliot) [inline src/localauthentication.cs]: same (version to check) and `onlyOn64 : true`

**Comment by dalexsoto** [comment]
Fixed attribute, verified and it does not exist in Mac 10,12,3

**Comment by dalexsoto** [comment]
monotouch-test - https://bugzilla.xamarin.com/show_bug.cgi?id=51800

**Pull request merged**